net/unbound/files/unbound.sh

```diff
@@ -16,7 +16,9 @@
 #
 # This builds the basic UCI components currently supported for Unbound. It is
 # intentionally NOT comprehensive and bundles a lot of options. The UCI is to
-# be a simpler presentation of the total Unbound conf set.
+# be a simpler presentation of the total Unbound conf set. However, it does
+# include support for the majority of OpenWrt/LEDE original implementation of
+# dnsmasq options. It gets more complicated with this.
```

Comment:
The original plan for simple UCI is clearly busted. I mashed some text in there to indicate this. It still needs "word smithing."